ice.c

```diff
@@ -2023,6 +2025,44 @@ static void janus_ice_cb_nice_recv(NiceAgent *agent, guint stream_id, guint comp
 							stream->video_is_keyframe = &janus_h264_is_keyframe;
 					}
 				}
+				/* If we need to handle transport wide cc*/
+				if (stream->do_transport_wide_cc) 
```

Comment:
Please put the `{` on the same line as the condition (just a nit to match the coding style of the rest of the code). There are a few other occurences in the code too.